cef/DemoWindow.rbfrm

```diff
@@ -3099,7 +3099,7 @@ End
 	#tag Event
 		Sub Action()
 		  testField.IndentVisually = me.Value
-		  testField.ClearDirtyLines
+		  'testField.ClearDirtyLines
```

Comment:
Yeah, but please explain why you comment something out, then, next time. I guess you commented it out to test that changing indentation doesn't clear the dirty markers, right? I really don't know what all the code in the demo window does. Keep it mind that most of it wasn't even written by me. Just remove the line entirely.